### PR TITLE
Adding vSphere,Azure Windows worklflows to chat-bot

### PR DESCRIPTION
**Pull request by rrasouli**

### CODE REVIEW

File: core-services/ci-chat-bot/workflows-config.yaml [hunk 1]
```diff
@@ -312,6 +312,13 @@ workflows:
     platform: vsphere
   cucushift-installer-rehearse-vsphere-ipi-sdn:
     platform: vsphere
+  cucushift-installer-rehearse-vsphere-ipi-ovn-winc:
```
Comment (liangxia) [inline core-services/ci-chat-bot/workflows-config.yaml:315]: Would you mind to sort them in alphanumeric order ?

File: core-services/ci-chat-bot/workflows-config.yaml [hunk 2]
```diff
@@ -168,6 +168,14 @@ workflows:
     platform: aws
   cucushift-installer-rehearse-azure-ipi:
     platform: azure
+  cucushift-installer-rehearse-azure-ipi-ovn-winc:
+        tag: upi-installer
```
Comment (liangxia) [inline core-services/ci-chat-bot/workflows-config.yaml:172]: Looks like this line is unexpected

Comment (rrasouli) [reply core-services/ci-chat-bot/workflows-config.yaml:172]: Apologies for that... Fixing and pushing

**Pull request merged**